### PR TITLE
Added 5 new units

### PR DESCRIPTION
**Pull request by chrlaney**
CentiM3-PER-CentiM3, GM-PER-HectoGM, MicroGM-PER-L, MicroS-PER-CM, MicroS-PER-CM2. Would appreciate a look over prior to submission to QUDT @mobb @jhp7e. Have 6 more to add if I'm going the right direction.

### CODE REVIEW

**Comment by steveraysteveray** [comment]
1. Each of your unit declarations is missing a "." at the end of the last triple.
2. For better or worse, we are using the American convention of "0.001-fold....." rather than "0,001-fold"

Once you fix the first problem, I will more easily be able to send it to the validator.

Thanks for all this!

**Comment by steveraysteveray** [comment]
After another quick look, I found various syntactical errors - missing " in descriptions, missing ; at the ends of lines, things like that. I recommend you try opening your file in an editor that understands Turtle (I use Visual Studio with the very nice Turtle Language Server extension by Stardog Union). That will catch all these syntax errors.

**Comment by steveraysteveray** [comment]
Here's a screenshot of validation errors after the syntax issues. A few explanations:
- the qudt:plainTextDescription should not have a language tag
- the first three units are probably duplicates of what is already in the QUDT vocabulary (haven't checked)
- you will need to define any nonexistent dimension vectors as needed, in the dimension vector file
- for the last two, we ask that the unit refer to all quantity kinds that are qudt:exactMatch with each other, so they need pointers to LinearVelocity as well
image

**Comment by jhp7e** [comment]
@chrlaney I think some of your descriptions are off by a decimal place.  GM is  "0.001-fold of the SI base unit kilogram" rather than "0.0001-fold of the SI base unit kilogram"  (a gram is 1/1,000 of kg, not 1/10,000).... and that may affect the multipliers as well.  For example, I think GM-PER-M2-HR  with base units KiloGM, Meter and SEC  the multiplier would be .001/(1*3600) =  2.77777E-7 or 0.00000027777778.  Similarly, for MicroGM-PER-GM-HR I get 2.77778E-10, not 2.7777 .....

Although I'd agree with you on the dimension vector for MicroGM-PER-GM-HR, to be consistent with other similar units I'd use: qkdv:A0E0L0I0M0H0T0D1   for some reason they don't seem to use D1 with anything else in the unit vector, although such vectors exist in the dimension vectors list.  

But apart from @steveraysteveray's excellent comments, they look pretty good to me otherwise.  Forge on!

**Comment by mobb** [comment]
I'll fix the formatting problems and add vectors as needed.

File: QUDTcontributions2023.ttl [hunk 1]
```diff
@@ -54,6 +54,102 @@ unit:KiloM2
...
+  qudt:plainTextDescription "0,001-fold of the SI base unit kilogram divided by 0,1-fold of the SI base unit kilogram" ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.01 ;
+  qudt:expression "\\(g-hg^{-1}\\)"^^qudt:LatexString ;
+  qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T0D0 ;  
+  qudt:qkdvNumerator qkdv:A0E0L0I0M1H0T0D0 ;
+  qudt:qkdvDenominator qkdv:A0E0L0I0M-1H0T0D0 ;
```
Comment (steveraysteveray) [inline QUDTcontributions2023.ttl:69]: This denominator should also be M1, not M-1. This way, when you divide the numerator by the denominator, they cancel.

File: QUDTcontributions2023.ttl [hunk 2]
```diff
@@ -54,6 +54,102 @@ unit:KiloM2
...
+  a qudt:Unit ;
+  dcterms:description "0,001-fold of the SI base unit kilogram divided by 0,1-fold of the SI base unit kilogram" ;
+  qudt:plainTextDescription "0,001-fold of the SI base unit kilogram divided by 0,1-fold of the SI base unit kilogram" ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.01 ;
+  qudt:expression "\\(g-hg^{-1}\\)"^^qudt:LatexString ;
+  qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T0D0 ;
```
Comment (steveraysteveray) [inline QUDTcontributions2023.ttl:67]: This should be D1.

**Comment by chrlaney** [comment]
Thanks @mobb ! I have been busy with other work this week, but really appreciated feedback/help from @steveraysteveray and @jhp7e. I installed the turtle editor suggested by @steveraysteveray and will incorporate lessons learned into a few more units that I need to add. I did check in QUDT to make sure I wasn't duplicating any that already existed, but if I missed them please let me know so I can improve my searching!

File: QUDTcontributions2023.ttl [hunk 3]
```diff
@@ -54,6 +54,102 @@ unit:KiloM2
...
+  rdfs:label "Gram Per Hectogram"@en ;
+
+unit:CentiM3-PER-CentiM3
+  a qudt:Unit ;
+  dcterms:description "A common metric unit of volume. One cubic centimeter equals exactly 0.000001 cubic meters." ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.000001 ;
```
Comment (steveraysteveray) [inline QUDTcontributions2023.ttl:85]: The ratio is equal to the ratio of M3-PER-M3, thus the conversion multiplier should be 1.0

File: QUDTcontributions2023.ttl [hunk 4]
```diff
@@ -54,6 +54,102 @@ unit:KiloM2
...
+  qudt:expression "\\(cm^{3}-cm^{-3}\\)"^^qudt:LatexString ;
+  qudt:qkdvNumerator qkdv:A0E0L3I0M0H0T0D0 ;
+  qudt:qkdvDenominator qkdv:A0E0L-3I0M0H0T0D0 ;
```
Comment (steveraysteveray) [inline QUDTcontributions2023.ttl:88]: L3, not L-3. Also, you need qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T0D1 ;

File: QUDTcontributions2023.ttl [hunk 5]
```diff
@@ -54,6 +54,102 @@ unit:KiloM2
...
+  a qudt:Unit ;
+  dcterms:description "0,000000001-fold of the SI base unit kilogram divided by the unit litre" ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.000000001 ;
+  qudt:expression "\\(cm^{3}-cm^{-3}\\)"^^qudt:LatexString ;
+  qudt:qkdvNumerator qkdv:A0E0L0I0M1H0T0D0 ;
+  qudt:qkdvDenominator qkdv:A0E0L-3I0M0H0T0D0 ;
```
Comment (steveraysteveray) [inline QUDTcontributions2023.ttl:106]: This should be L3, not L-3, if you choose to make the numerator and denominator explicit. Either way, you still need qudt:hasDimensionVector qkdv:A0E0L-3I0M1H0T0D0 ;

File: QUDTcontributions2023.ttl [hunk 6]
```diff
@@ -54,6 +54,102 @@ unit:KiloM2
...
+  a qudt:Unit ;
+  dcterms:description "0,000001-fold of the SI derived unit siemens divided by 0,01-fold of the SI base unit meter" ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.0001 ;
+  qudt:expression "\\(uS-cm^{-1}\\)"^^qudt:LatexString ;
+  qudt:qkdvNumerator qkdv:A0E2L-2I0M-1H0T3D0 ;
+  qudt:qkdvDenominator qkdv:A0E0L-1I0M0H0T0D0 ;
```
Comment (steveraysteveray) [inline QUDTcontributions2023.ttl:125]: Same issues as with the unit above.

File: QUDTcontributions2023.ttl [hunk 7]
```diff
@@ -54,6 +54,102 @@ unit:KiloM2
...
+  a qudt:Unit ;
+  dcterms:description "0,000001-fold of the SI derived unit siemens divided by the squared 0,01-fold of the SI base unit meter" ;
+  qudt:applicableSystem sou:CGS ;
+  qudt:applicableSystem sou:CGS-EMU ;
+  qudt:applicableSystem sou:CGS-GAUSS ;
+  qudt:applicableSystem sou:SI ;
+  qudt:conversionMultiplier 0.01 ;
+  qudt:expression "\\(uS-cm^{-2}\\)"^^qudt:LatexString ;
+  qudt:qkdvNumerator qkdv:A0E2L-2I0M-1H0T3D0 ;
+  qudt:qkdvDenominator qkdv:A0E0L-2I0M0H0T0D0 ;
```
Comment (steveraysteveray) [inline QUDTcontributions2023.ttl:144]: Same issues as above.

**Pull request merged**